### PR TITLE
Update types/node and eslint-plugin-import

### PR DESCRIPTION
**Pull request by vsvipul**
Update types/node and eslint-plugin-import to suppress dependabot alerts.

### CODE REVIEW

File: dist/restore/index.js [hunk 1]
```diff
@@ -37903,9 +37903,17 @@ AbortError.prototype = Object.create(Error.prototype);
 AbortError.prototype.constructor = AbortError;
 AbortError.prototype.name = 'AbortError';
 
+const URL$1 = Url.URL || whatwgUrl.URL;
```
Comment (bishal-pdMSFT) [inline dist/restore/index.js:37906]: Did node type update introduced these save and restore js file changes?

Comment (vsvipul) [reply dist/restore/index.js:37906]: No @bishal-pdMSFT . I suppose the .js files in main are out of date. I've updated those as well here by building those, as it was one of the tests.

**Comment by vsvipul** [comment]
Closing as this generates extra .js file changes. Don't know the root cause for this yet.

**Comment by Soundpisey** [comment]
I have seen it. Thank you very much.

**Pull request closed without merging**